src/Symfony/Bridge/Twig/Node/SearchAndRenderBlockNode.php

```diff
@@ -49,8 +50,14 @@ public function compile(Compiler $compiler): void
                         // of variables at compile time.
                         $labelIsExpression = false;
 
+                        if (method_exists(CoreExtension::class, 'testEmpty')) {
```

Comment:
```suggestion
                        // twig_test_empty is deprecated in Twig 3.9
                        if (method_exists(CoreExtension::class, 'testEmpty')) {
```
Adding such comment helps when we remove compatibility with older versions years after.

Reply:
add comments (slightly reworded)